manifests/mod/proxy.pp

```diff
@@ -1,33 +1,34 @@
 # @summary
 #   Installs and configures `mod_proxy`.
-# 
+#
 # @param proxy_requests
 #   Enables forward (standard) proxy requests.
-# 
+#
 # @param allow_from
 #   List of IPs allowed to access proxy.
-# 
+#
 # @param apache_version
 #   Used to verify that the Apache version you have requested is compatible with the module.
-# 
+#
 # @param package_name
 #   Name of the proxy package to install.
-# 
+#
 # @param proxy_via
 #   Set local IP address for outgoing proxy connections.
-# 
+#
 # @param proxy_timeout
 #   Network timeout for proxied requests.
-# 
+#
 # @see https://httpd.apache.org/docs/current/mod/mod_proxy.html for additional documentation.
 #
```

Comment:
Could you please add the new parameter information

Reply:
Done